packages/netlify-cms-core/src/formats/frontmatter.ts

```diff
@@ -27,15 +27,16 @@ const parsers = {
     parse: (input: string) => {
       let JSONinput = input.trim();
       // Fix JSON if leading and trailing brackets were trimmed.
-      if (JSONinput.substr(0, 1) !== '{') {
+      if (JSONinput.slice(0, 1) !== '{') {
         JSONinput = '{' + JSONinput + '}';
       }
       return jsonFormatter.fromFile(JSONinput);
     },
     stringify: (metadata: object) => {
       let JSONoutput = jsonFormatter.toFile(metadata).trim();
       // Trim leading and trailing brackets.
-      if (JSONoutput.substr(0, 1) === '{' && JSONoutput.substr(-1) === '}') {
+      if (JSONoutput.slice(0, 1) === '{' && JSONoutput.slice(-1) === '}') {
+        // eslint-disable-next-line unicorn/prefer-string-slice
         JSONoutput = JSONoutput.substring(1, JSONoutput.length - 1);
```

Comment:
JSONoutput = JSONoutput.substring(1, JSONoutput.length - 1);
can be
JSONoutput = JSONoutput.slice(1, -1);